components/MashTags/MashTags.tsx

```diff
@@ -0,0 +1,90 @@
+import moment from 'moment';
+
+interface Props {
+  filter: string;
+  createdAt: string;
+  initialDecision: string | undefined;
+}
+
+const MashTags = ({
+  filter,
+  createdAt,
+  initialDecision,
+}: Props): React.ReactElement => {
+  const currenttime = new Date().toISOString();
+  const hoursPassed = moment(currenttime, 'YYYY-MM-DD[T]HH:mm:ss. SSS[Z]').diff(
+    moment(createdAt, 'YYYY-MM-DD[T]HH:mm:ss. SSS[Z]'),
+    'hours'
+  );
+  const minsPassed = moment(currenttime, 'YYYY-MM-DD[T]HH:mm:ss. SSS[Z]').diff(
+    moment(createdAt, 'YYYY-MM-DD[T]HH:mm:ss. SSS[Z]'),
+    'minutes'
+  );
```

Comment:
```
const currentTime = new Date();
  const createdAtTime = new Date(createdAt);

  const hoursPassed = currentTime.getHours() - createdAtTime.getHours();
  const minsPassed = currentTime.getMinutes() - createdAtTime.getMinutes();
```

I'm actually slightly confused how using moment was working considering we don't have it as an installed dependency

Reply:
In my opinion calling moment and having it parse iso formatted date strings is adding complexity that we don't need to find out how many hours/minutes have passed. (explanation for the above comment with suggested code change.